app.js

```diff
@@ -14,12 +14,33 @@ app.use(express.static(path.join(__dirname, 'public')));
 
 // Register the location for handlebars partials here:
 
-// ...
+hbs.registerPartials(path.join(__dirname, 'views/partials'));
 
 // Add the route handlers here:
 
 app.get('/', (req, res) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res) => {
+  punkAPI
+    .getBeers()
+    .then(beersFromApi => {
+      res.render('beers', { beers: beersFromApi });
+    })
+
+    .catch(error => console.log(error));
+});
+
+app.get('/random-beer', (req, res) => {
+  punkAPI
+    .getRandom()
+    .then(responseFromAPI => {
+      console.log(responseFromAPI.name);
+      res.render('random-beer', { randomBeer: responseFromAPI });
```

Comment:
This is correct. Another way to do it is to pass` responseFromAPI[0] `to res.render. This is because the `.getRandom() `always return an array of one element and so we can skip having to do the loop in the hbs.